aws/request/http_request_1_4.go

```diff
@@ -0,0 +1,21 @@
+// +build !go1.5
+
+package request
+
+import "net/http"
+
+func copyHTTPRequest(r *http.Request) *http.Request {
```

Comment:
Suggest passing body in instead of coping `r.Body` value. also prevents need setting it outside of copy function.